pom.xml

```diff
@@ -119,6 +128,19 @@
       <version>2.2.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>cloudbees-folder</artifactId>
```

Comment:
Fixed the misaligned indent
https://github.com/jenkinsci/credentials-plugin/blob/37841b8970cfc856d6463739b86fffe4792944dd/pom.xml#L116-L127